setup/gcp/scripts/run_forseti.sh

```diff
@@ -36,12 +36,33 @@ sudo systemctl restart forseti.service
 # Wait until the service is started
 sleep 10s
 
+# Set the output format to json
+forseti config format json
+
 # Run inventory command
 MODEL_NAME=$(/bin/date -u +%Y%m%dT%H%M%S)
 echo "Running Forseti inventory."
 forseti inventory create --import_as ${MODEL_NAME}
 echo "Finished running Forseti inventory."
-sleep 10s
+sleep 5s
+
+LOOP_COUNT=0
+MAX_LOOP_COUNT=50
+# Wait until the model is initialized
```

Comment:
Instead of `is initialized`, it would be nice to be more clear that we are waiting for models to be finished building.